fem/fespace.hpp

```diff
@@ -351,6 +399,10 @@ class FiniteElementSpace
    void BuildBdrElementToDofTable() const;
    void BuildFaceToDofTable() const;
 
+   void GetEssentialBdrEdgesFaces(const Array<int> &bdr_attr_is_ess,
```

Comment:
Document.

Reply:
Done in fc63c5e90b5ace705c64319123d3715d1dbe250f